(Cosmetic) Made case type more consistent

PhoenixWhitefire opened the pull request:
The naming of the modules are in PascalCase, but the inside of the scripts are written in camelCase. Luau's official styleguide also states Luau should be written in PascalCase for consistency. Also, the underscore can hurt readability, someone would expect an underscore to be in-between two words, but you've used it here to prevent Roblox replacing the error message with the global "error" function.

PhoenixWhitefire committed:
(Cosmetic) Made case type more consistent

The naming of the modules are in PascalCase, but the inside of the scripts are written in camelCase. Luau's official styleguide also states Luau should be written in PascalCase for consistency. Also, the underscore can hurt readability, someone would expect an underscore to be in-between two words, but you've used it here to prevent Roblox replacing the error message with the global "error" function.

lucasmz-dev commented:
Uh would you mind linking the style guide, I can't find it? I only know evaera's.

I believe most people prefer to have local cases in whatever indentation they are, so if they're on the top indentation it's PascalCase, and on anything that's indented, it's camelCase;

lucasmz-dev commented:
Oh, and btw! In this case, the right thing to use would be "errorMessage" as it better describes it.

PhoenixWhitefire committed:
Error -> ErrorMessage

More descriptive (suggestion by LucasMZ)

PhoenixWhitefire committed:
tokens -> Tokens

Accidental typo, code defines "Tokens" but later uses "tokens" with a lowercase "t".

PhoenixWhitefire commented:
@LucasMZReal https://roblox.github.io/lua-style-guide/

lucasmz-dev commented:
@PhoenixWhitefire ?
![image](https://user-images.githubusercontent.com/55422065/147044782-1312b69d-de05-40ed-a167-b5a8a8658992.png)

They specify using camelCase for locals (basically the ones that aren't on the top 'layer' of the code)
